wallet/src/main/resources/walletdb/migration/V3__wallet_db_add_dlcs.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE IF NOT EXISTS "wallet_dlcs" ("eventId" VARCHAR(254) PRIMARY KEY NOT NULL,"oracleInfo" VARCHAR(254) NOT NULL,"contractInfo" VARCHAR(254) NOT NULL,"timeouts" VARCHAR(254) NOT NULL,"isInitiator" INTEGER NOT NULL,"remoteExtPubKey" VARCHAR(254),"input" INTEGER NOT NULL,"remoteInput" INTEGER,"fundingInputs" VARCHAR(254) NOT NULL,"remoteFundingInputs" VARCHAR(254), "feeRate" VARCHAR(254),"changeSPK" VARCHAR(254) NOT NULL,"remoteChangeSPK" VARCHAR(254), "network" VARCHAR(254), "cetSigs" VARCHAR(254), "fundingSigs" VARCHAR(254), "oracleSig" VARCHAR(254));
```

Comment:
@rorp has issues with the `IF NOT EXISTS` part, he thinks it should just be `CREATE TABLE .. `